tasks/share_task_qc_atac.wdl

```diff
@@ -20,11 +20,11 @@ task qc_atac {
         File raw_bam_index
         File filtered_bam
         File filtered_bam_index
-        File? tss
+        File tss
         String genome_name
         String? prefix
-        String docker_image = "us.gcr.io/buenrostro-share-seq/share_task_qc_atac"
-
+        #String docker_image = "us.gcr.io/buenrostro-share-seq/share_task_qc_atac"
+        String docker_image = "mknudson/share_task_qc_atac:test"
```

Comment:
We don't need this because the docker will be automatically updated since you are pushing a new one as well